streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -204,6 +207,15 @@ private void logRestoreOffsets(final TopicPartition partition, final long starti
     }
 
     @Override
+    public void refreshChangelogInfo() {
+        try {
+            partitionInfo.putAll(consumer.listTopics());
+        } catch (final TimeoutException e) {
+            log.debug("{} Could not fetch topic metadata; will fall back to partition by partition fetching", logPrefix);
```

Comment:
Is this actually true? I don't see where we do partition by partition anymore

Reply:
My bad, forgot to update the log4j entry.